ztp/gitops-subscriptions/Makefile

```diff
@@ -45,4 +49,41 @@ gen-files: build $(KUSTOMIZE)
 	$(KUSTOMIZE) build --enable-alpha-plugins $(ACM_POLICYGEN_EX_DIR) -o $(ACM_POLICYGEN_EX_DIR)/out/
 
 clean:
-	rm -rf $(ACM_POLICYGEN_KUSTOMIZE_DIR) $(POLICYGEN_KUSTOMIZE_DIR) $(POLICYGEN_EX_DIR)/out $(ACM_POLICYGEN_EX_DIR)/out $(ACM_POLICYGEN_EX_DIR)/source-crs $(POLICYGEN_EX_DIR)/source-crs
+	rm -rf $(ACM_POLICYGEN_KUSTOMIZE_DIR) $(POLICYGEN_KUSTOMIZE_DIR) $(POLICYGEN_EX_DIR)/out $(ACM_POLICYGEN_EX_DIR)/out $(ACM_POLICYGEN_EX_DIR)/source-crs $(POLICYGEN_EX_DIR)/source-crs $(ACM_POLICYGEN_FROM_PGT_DIR)
+
+pgt2acm-clean:
+	rm -rf ../tools/pgt2acm
+
+pgt2acm-get: pgt2acm-clean
```

Comment:
If the tool is moving under ztp/tools, is there any need to clone it?

Reply:
Fixed. I was planning to copy the pgt2acm project from the original repo and periodically synchronize with a script to avoid issues with having multiple go projects under the same Github repo.  I removed all that synchronization logic.